### ISSUE TITLE
Android building issue using updated code.

### ISSUE THREAD

**Issue by AlexB123** [syzygy1/Cfish#202]
Hello, i have a problem with android compilation (using Clang on Termux, and cross compiling with NDK, also clang). Source code taken from here https://github.com/Sopel97/Cfish/commit/666f0f87e2a339f499ca1e5e5d49a9a29d8cf074
When i'm trying to compile engine for android i'm receiving the following errors, (see txt documents). Can you please take a look at this code? Something is very wrong there, and you are the only person who can fix it. :)
[Termux ERRORS.zip](https://github.com/syzygy1/Cfish/files/6804462/Termux.ERRORS.zip)
[NDK errors.zip](https://github.com/syzygy1/Cfish/files/6804463/NDK.errors.zip)

**Comment by JavaMast** [comment]
@AlexB123 
did you build it with sparse=yes ?
sparse has not working

Try again with updated SRCs
 https://github.com/Sopel97/Cfish/tree/update

**Comment by AlexB123** [comment]
> 
> 
> @AlexB123
> did you build it with sparse=yes ?
> sparse has not working
> 
> Try again with updated SRCs
> https://github.com/Sopel97/Cfish/tree/update

Yes, with previous source code, i've tried sparse=yes and sparse=no, didn't work.

**Comment by AlexB123** [comment]
With last source code https://github.com/Sopel97/Cfish/commit/4c71e9dc631f497c9ed4d7e1f07a507f141c18d1
i'm getting less errors, which (i suppose) has to do with LTO. Again, with sparse=yes and sparse=no, i'm getting exactly the same errors.
![!with or without sparce](https://user-images.githubusercontent.com/19660133/125519423-17b09062-6ee2-47b8-a7e3-482387f731a3.jpg)

By turning lto=no, i'm getting the following errors, something is still broken.
![!without lto](https://user-images.githubusercontent.com/19660133/125519628-1f521966-4903-491a-8821-f2f100cc890e.jpg)

**Comment by SwiftSnips** [comment]
Have you gotten anywhere on Termux Alex? I got it down to 1 just 1 error, that being the vec_zero_psqt error.

**Comment by AlexB123** [comment]
> 
> 
> Have you gotten anywhere on Termux Alex? I got it down to just 1 error, that being the vec_zero_psqt error.

Nope, without the intervention of an expert, it is unlikely that it will work.

**Comment by SwiftSnips** [comment]
Sopel fixed the one issue where 32x4 was backwards but im not sure he knows what to do about the vec_zero_psqt problem

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
